data/campaigns/Liberty/scenarios/05_Hide_and_Seek.cfg

```diff
@@ -586,6 +586,30 @@ Been a tough few days since we left Elensefar with all them patrols running arou
         )}
     [/event]
 
+    [event]
+        name=die
+        [filter]
+            type=Iron Mauler
+        [/filter]
+        [filter_second]
+            id=Harper
+        [/filter_second]
+
+        [message]
+            speaker=Harper
+            message= _ "Uncle, we did it! We beat one of them iron monsters!"
```

Comment:
This should have a `# po:` comment, because the translators won't have the benefit of seeing that it's in an iron mauler's die event.